### PR TITLE
feat(basket): #CRRE-462 redirect establishment requests when you are a validator

### PR DESCRIPTION
**Pull request by BastienBenard**
## Describe your changes
To avoid cases where the validators are prescribers, make a basket sending for validation and stop there, automatically redirect them to the table of pending requests on the tab side with a pop up to alert them that it is necessary to send all in the region.

The core tools do not manage the muti-establishment for the rights, it was decided to add an API which allows to restore the rights of the user according to the structure.
For our case, the condition is "If the user is a validator, I redirect him".
Let's take a user who has two establishments:

- "id_etablisement_1"
- "id_etabisement_2"

On establishment 1 I have the right CRRE.VALIDATOR
On establishment 2 I do not have the CRRE.VALIDATOR right

With the model.me.hasRight(CRRE.VALIDATOR) this will return True because our user is in at least 1 structure where he has the CRRE.VALIDATOR right.
With this dev we will be able to do:

- hasRight(CRRE.VALIDATOR, "id_etablisement_1") => true
- hasRight(CRRE.VALIDATOR, "id_etablisement_2") => false

Doc API: https://confluence.support-ent.fr/display/CRRE/API+1.4.0

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/i18n/fr.json [hunk 1]
```diff
@@ -837,5 +837,6 @@
   "crre.timeline.validator.partially.send": "a été partiellement transmise au libraire.",
   "crre.timeline.validator.valid": "a été validée par la Région Île-de-France.",
   "crre.timeline.validator.partially.valid": "a été partiellement validée par la Région Île-de-France.",
-  "crre.timeline.validator.refused": "a été refusée par la Région Île-de-France."
+  "crre.timeline.validator.refused": "a été refusée par la Région Île-de-France.",
+  "crre.prescriptor.confirm.basket.massage": "Votre panier va être soumis au valideur de l'établissement."
```
Comment (flmartineau) [inline src/main/resources/i18n/fr.json:841]: massage ? ^^'

File: src/main/java/fr/openent/crre/controllers/WorkflowController.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import java.util.stream.Collectors;
+
+public class WorkflowController extends ControllerHelper {
+    private final WorkflowService workflowService;
+
+    public WorkflowController(ServiceFactory serviceFactory) {
+        this.workflowService = serviceFactory.getWorkflowService();
+    }
+
+    @Post("/user/workflow")
```
Comment (sjollois) [inline src/main/java/fr/openent/crre/controllers/WorkflowController.java:28]: pk un Post ?

**Review by sjollois — APPROVED** [review]

File: src/main/resources/i18n/fr.json (hunk 1 above)
Comment (BastienBenard) [reply src/main/resources/i18n/fr.json:841]: oups ...

File: src/main/java/fr/openent/crre/controllers/WorkflowController.java (hunk 2 above)
Comment (BastienBenard) [reply src/main/java/fr/openent/crre/controllers/WorkflowController.java:28]: Parce que il peut avoir une grande quantité de structure. Donc si on met les paramètres en url cela risque d'être trop long. Et le get de axios ne gère pas les paramètre en body.

**Review by qpic-cgi — APPROVED** [review]

**Pull request merged**